packages/shared/friends/sagas.ts

```diff
@@ -1164,6 +1198,18 @@ function* handleUpdateFriendship({ payload, meta }: UpdateFriendship) {
           totalReceivedRequests: updateTotalFriendRequestsPayload.totalReceivedRequests + 1
         }
 
+        // We only send this message in the new flow
+        if (newFriendRequestFlow) {
```

Comment:
Let's add a `TODO` for all these conditionals to remember removing this in the near future.